experimental/ratify/proto/v1/common.proto

```diff
@@ -0,0 +1,33 @@
+syntax="proto3";
+
+package common;
+
+option go_package = "github.com/deislabs/ratify/experimental/proto/v1/common";
+
+message ArtifactDescriptor {
+    message Attributes {
+        map<string,string> values = 1;
+    }
+    string path = 1;
```

Comment:
just curiuous: what is the purpose of the path variable for a Descriptor?

Reply:
the thought was to there would be value in have 1 "known" property rather than just a dynamic collection